commands/create/cluster/v5.go

```diff
@@ -165,6 +185,13 @@ func addClusterV5(def *types.ClusterDefinitionV5, args Arguments, clientWrapper
 		}
 
 		nodePoolRequestBody := &models.V5AddNodePoolRequest{}
+
+		if response.Payload.MasterNodes != nil && len(response.Payload.MasterNodes.AvailabilityZones) < 1 {
```

Comment:
If the created cluster has no AZs, then we also create the node pool with no AZs